### PR TITLE
Update user documentation structure

### PR DESCRIPTION
**Pull request by thebe14**
# Summary

The documentation needs some changes to the structure, in order to accommodate new services and to make it easier to navigate. A lot of (most) content from the EGI-ACE [D2.3](https://docs.google.com/document/d/1ycq076_SQ6pIH3MabruDpCviIERWVQ_l/edit#) and [D2.5](https://docs.google.com/document/d/1ycq076_SQ6pIH3MabruDpCviIERWVQ_l/edit#) should end up in our user documentation.

- Added the necessary new pages
- Provided content for the new pages
- Revised content of existing pages to bring them in line with the rest of the documentation style.
---

**Related issue :** Implements #411

### CODE REVIEW

**Comment by andrea-manzi** [comment]
great work! let's make the linter happy and then we can have a preview and review the changes

**Review by gwarf — COMMENTED** [review]
Thanks.
Added a few comments, mainly letting other service owners review the content for their services.
Once it lints well we can enable the preview.

File: content/en/users/compute/orchestration/im/_index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+and update, and monitoring of virtual infrastructures. 
+
+IM is integrated with the [EGI Check-in Service](../../../aai/check-in) and supports
+a wide variety of backends, either public (such as
+[Amazon Web Services](https://aws.amazon.com/),
+[Google Cloud](http://cloud.google.com/) or
+[Microsoft Azure](http://azure.microsoft.com/)) or on-premises (such as
+[OpenStack](http://www.openstack.org/)) or federated infrastructures (such as 
+[EGI Cloud Compute](../../cloud-compute)),
+thus making user applications cloud agnostic.
```
[the remark this answers is not in the document]
Comment (thebe14) [reply content/en/users/compute/orchestration/im/_index.md:24]: Agree.

File: content/en/users/aai/check-in/_index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+linkTitle: Check-in
+type: docs
+weight: 30
+description: >
+  Control access to resources in the EGI Cloud
+---
+
+## What is it?
+
+[EGI Check-in](https://www.egi.eu/services/check-in/) is a web service that
```
[the remark this answers is not in the document]
Comment (thebe14) [reply content/en/users/aai/check-in/_index.md:12]: Updated

**Comment by thebe14** [comment]
> i see one problem now...the links to the doc we have added to the D2.5 deliverables will be incorrect ..so we should revise them ASAP when we approve this PR

EGI-ACE D2.5 updated

**Comment by gwarf** [comment]
Looks good!
Some quick/first feedback: I'm not sure at all about https://docs.egi.eu/documentation/414/users/getting-started/task-force/ like about how it's named, where it's in the menu.
The FedCloud Task Force is only a very specific task force, focusing on mainly the cloud activities. I would find this better in the Compute part, or otherwise in something named/organised differently.
Same for the CLI in the getting started, it would like it's a CLI for all EGI services, while it's certainly not the case

**Comment by thebe14** [comment]
> I'm not sure at all about https://docs.egi.eu/documentation/414/users/getting-started/task-force/ like about how it's named, where it's in the menu. The FedCloud Task Force is only a very specific task force, focusing on mainly the cloud activities. I would find this better in the Compute part, or otherwise in something named/organised differently.

That will get improved when the Getting Started part gets revised next. A lot of stuff about how to request access, the architecture, and more from D2.3 and D2.5 need to be added. It is not what this PR should fix. 

> Same for the CLI in the getting started, it would like it's a CLI for all EGI services, while it's certainly not the case

The CLI is more or less "the CLI" for EGI, a lot of docs pages refer to it (there were many places that were introducing fedcloudcli, now all those refer to `/users/getting-started/cli`). Some services have additonal CLIs as well, but the docs for those services will mention them. And fedcloudcli is and will be improved to cover more and more of the EGI services. Finally, improving this is not the purpose of this PR.

**Comment by thebe14** [comment]
@andrea-manzi I did a lot of changes in the top pages for the data services. uniformized titles and subtitles (e.g. no links or acronyms in titles), added a _What is it?_ paragraph to each. please review them, let me know if it's good (or you can improve it further in future PRs).

Data TODOs:

- the openRDM page has to be filled in
- some diagrams from D2.5 could also make their way into the docs

File: content/en/users/data/management/_index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## Specialized data management
+
+The following specialized data management services are also available:
+
+- [EGI Data Transfer](data-transfer) is a low-level service to move data from
+  one [Grid](../storage/grid-storage) or [Object](../storage/object-storage) storage
+  to another. It is used internally by [Rucio](rucio) to schedule transfers based
+  on the data policies defined by the users.
+- [openRDM](open-rdm) is a combined [FAIR](https://en.wikipedia.org/wiki/FAIR_data)
```
Comment (andrea-manzi) [inline content/en/users/data/management/_index.md:46]: As we don't have a doc for now i would like the EOSC marketpace entry https://marketplace.eosc-portal.eu/services/openrdm-eu

File: content/en/users/security/_index.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+type: docs
+draft: true
+weight: 60
+description: >
+  Security services in the EGI Cloud
+---
+
+## Using cloud services securely
+
+Cloud security in the EGI Federated Cloud is the highest priority.
```
Comment (andrea-manzi) [inline content/en/users/security/_index.md:13]: I think we should not refer only to cloud here, as the service could be used also by other services in the EGI infra

File: content/en/users/data/management/datahub/_index.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+EGI DataHub supports multiple access policies:
+
+- Unauthenticated, open access
+- Access after user registration or
+- Access restricted to members of a [Virtual Organization](../../../aai/check-in//vos) (VO)
+
+Data replication in EGI DataHub may take place either on-­demand or automatically.
+Replication uses a file catalogue to enable tracking of logical and physical copies of data.
+
+{{% alert title="Note" color="info" %}} EGI DataHub is based on
```
Comment (andrea-manzi) [inline content/en/users/data/management/datahub/_index.md:42]:
```suggestion
{{% alert title="Note" color="info" %}} EGI DataHub is based on the
```

File: content/en/users/data/management/datahub/_index.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+[compute facilities](../../../compute) they use, in order to exploit it efficiently.
+This is as simple as selecting which (subset of the) data should be available
+at which supporting provider.
+
+The main features of DataHub are:
+
+- Discovery of data spaces via a central portal
+- Policy based data access
+- Replication of data across providers for resiliency and availability purposes
+- Access to data from the [AppDB](../../../compute/cloud-compute/images/)
```
Comment (andrea-manzi) [inline content/en/users/data/management/datahub/_index.md:27]: at the the integration with AppDB has never been implemented, so given that we are revising this part i would remove it

File: content/en/users/data/management/_index.md (hunk 3 above)
Comment (thebe14) [reply content/en/users/data/management/_index.md:46]: why not use the 2-3 paragraphs from D2.5? and also link to the eosc marketplace

File: content/en/users/security/_index.md (hunk 4 above)
Comment (thebe14) [reply content/en/users/security/_index.md:13]: Fixed

File: content/en/users/data/management/datahub/_index.md (hunk 6 above)
Comment (thebe14) [reply content/en/users/data/management/datahub/_index.md:27]: Done

File: content/en/users/aai/_index.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+description: >
+  Identity and access management in the EGI Cloud
+---
+
+## Working with federated resources
+
+[EGI is a federation](https://www.egi.eu/federation/) of
+compute and storage resource providers.
+
+In order for the providers to be able to
```
Comment (enolfc) [inline content/en/users/aai/_index.md:15]: I would move this to active instead of passive form

File: content/en/users/aai/check-in/_index.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
+---
+title: EGI Check-in
+linkTitle: Check-in
+type: docs
+weight: 30
+description: >
+  Control access to resources in the EGI Cloud
```
Comment (enolfc) [inline content/en/users/aai/check-in/_index.md:7]:
```suggestion
  Control access to resources in the EGI Infrastructure
```
or something similar

File: content/en/users/compute/_index.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+EGI provides a variety of compute services for each of these use cases.
+Researchers can choose the best compute tool for each application, from
+customisable and elastic Virtual Machine (VM) based cloud to fully managed
+distributed platforms that will run their jobs using CPUs, GPUs or both.
+
+The compute services are summarized below:
+
+- [Cloud Compute](./cloud-compute/) means **VM-based computing with associated
+  storage**. It delivers a customisable platform where users have complete control
```
Comment (enolfc) [inline content/en/users/compute/_index.md:26]:
```suggestion
  storage**. It delivers a customisable computing resources where users have complete control
```

File: content/en/users/compute/_index.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+  computational problems using tightly coupled parallel processing: simulations,
+  analysis of large datasets or AI/ML workloads, usually relying on MPI for
+  interprocess communication.
+-->
+
+When researchers enounter the need large amounts of computing power, need hybrid
+solutions based on the above mentioned services, or simply do not care to manage the
+underlying compute facilities that will run their workloads,
+[cloud orchestrators](./orchestration/) can help provision, manage, and monitor
+compute resources, which then run scientific workloads.
```
Comment (enolfc) [inline content/en/users/compute/_index.md:53]: :+1: I like this one

File: content/en/users/compute/cloud-compute/automate/_index.md [hunk 11]
```diff
@@ -24,5 +24,5 @@ Automating this activity will help researchers to:
 - Easily share scientific pipelines with collaborators.
 - Make scientific applications cloud agnostic.
 
-To automate VM deployment, users can use any of the [cloud orchestrators](../../cloud-orchestration/)
+To automate VM deployment, users can use any of the [cloud orchestrators](../../orchestration)
```
Comment (enolfc) [inline content/en/users/compute/cloud-compute/automate/_index.md:27]:
```suggestion
To automate VM deployment, you can use any of the [cloud orchestrators](../../orchestration)
```

File: content/en/users/compute/cloud-compute/openstack/_index.md [hunk 12]
```diff
@@ -80,7 +80,7 @@ cat /etc/grid-security/certificates/*.pem >> $(python -m requests.certs)
 ## Authentication
```
Comment (enolfc) [inline content/en/users/compute/cloud-compute/openstack/_index.md:80]: Consider reorder this file: Put aai before OpenStack

File: content/en/users/compute/orchestration/_index.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+Infrastructure-as-a-Service (IaaS) Cloud resources. These tools have different
+levels of abstractions and features. See below to understand which to choose
+(or gets used automatically) in specific scenarios:
+
+<!-- markdownlint-disable line-length -->
+
+| Service Name                                 | Workload Type             | Use-Case |
+| -------------------------------------------- | ------------------------- | -------- |
+| [Infrastructure Manager](im)                 | VMs and containers        | Used to run workloads on a single IaaS Cloud provider. |
+| [Elastic Cloud Compute Cluster](ec3)         | VMs and containers        | Used when you need to run workloads on clusters that can be elastically scaled and potentially span more than one IaaS Cloud provider. |
```
Comment (enolfc) [inline content/en/users/compute/orchestration/_index.md:27]: Workload Manager missing here. It's probably not in the same category, but still needs to be mentioned

File: content/en/users/compute/orchestration/ec3/_index.md [hunk 14]
```diff
@@ -0,0 +1,39 @@
+---
+title: Elastic Cloud Compute Clusters
+linkTitle: Cloud Compute Clusters
```
Comment (enolfc) [inline content/en/users/compute/orchestration/ec3/_index.md:3]:
```suggestion
linkTitle: Elastic Cloud Compute Cluster
```

I think we should avoid dissolving the EC3 branding

File: content/en/users/compute/orchestration/im/cli/_index.md [hunk 15]
```diff
@@ -1,13 +1,12 @@
 ---
-title: "Infrastructure Manager Command-Line Interface"
-linkTitle: "Command Line"
+title: Infrastructure Manager Commands
```
Comment (enolfc) [inline content/en/users/compute/orchestration/im/cli/_index.md:2]:
```suggestion
title: Infrastructure Manager command-line interface
```

File: content/en/users/data/_index.md [hunk 16]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+## Working with data
+
+One of the main daily activities of researchers is to collect or create data
+that needs to be processed, analyzed, or shared. The EGI offering in this area
+is quite vast and includes both low level
+[storage services](./storage/) and
+[data management](./management/) frameworks, as well as specialized
+[data transfer](./management/data-transfer/) services.
```
Comment (enolfc) [inline content/en/users/data/_index.md:17]: Should we move data-transfer to stand outside of data management to keep this logic? @andrea-manzi

File: content/en/users/data/management/datahub/links/_index.md [hunk 17]
```diff
@@ -1,9 +1,10 @@
 ---
-title: "Links"
-linkTitle: "Links"
-type: docs
-description: "Related links"
+title: DataHub Links
```
Comment (enolfc) [inline content/en/users/data/management/datahub/links/_index.md:2]: I feel this page is not bringing much, although I foresee similar ones coming for other services as we improve the documentation. For sure I would not call it "Links", maybe something like "Extra information"?

File: content/en/users/data/management/rucio/_index.md [hunk 18]
```diff
@@ -1,31 +1,30 @@
...
 
 While Rucio is extremely scalable, the STFC Rucio Data Management Service is
 designed for smaller communities, with expected data needs up to tens of
 Petabytes. The fact that the underlying Rucio infrastructure is managed by STFC,
-allows communities to easily start using and or test Rucio with little setup
-cost required.
+allows communities to easily start using and/or test Rucio with little setup
+cost.
 
 ## Official Rucio Pages
```
Comment (enolfc) [inline content/en/users/data/management/rucio/_index.md:29]: Going back to the DataHub comment: this is similar to the "links" in DataHub, no? Maybe doing the same for DataHub, move the "Links" content to the overview DataHub page?

File: content/en/users/data/management/datahub/_index.md [hunk 19]
```diff
@@ -0,0 +1,118 @@
+---
```
Comment (enolfc) [inline content/en/users/data/management/datahub/_index.md:1]: This one is way larger than other services, feels a bit unbalanced. Are we expecting other services to have an overview page that grow this much?

File: content/en/users/data/storage/_index.md [hunk 20]
```diff
@@ -0,0 +1,39 @@
...
+  scenarios.
+- [Object Storage](object-storage) is persistent, hierarchical blob storage for
+  cloud native applications, archiving, or when data is shared between different
+  VMs or multiple steps of processing workflows.
+
+## Comparison of storage types
+
+The differences between Block, Grid, and Object Storage are summarized below:
+
+| Type       | Sharing                                                  | Accounting           | Usage                           |
```
Comment (enolfc) [inline content/en/users/data/storage/_index.md:33]: Update this with table 4.1 of D2.5

**Review by enolfc — CHANGES_REQUESTED** [review]
Good to see this @thebe14! 
Some (rather minor) comments here and there.

File: content/en/users/data/management/datahub/links/_index.md (hunk 17 above)
Comment (thebe14) [reply content/en/users/data/management/datahub/links/_index.md:2]: Agree. I left it there, for future improvement. This PR is already doing too much.

File: content/en/users/data/management/rucio/_index.md (hunk 18 above)
Comment (thebe14) [reply content/en/users/data/management/rucio/_index.md:29]: Yep. These should be be compacted to the absolute minimum, and the result either put as notes or as links on terms in the text. Future improvement, maybe not in this PR.

File: content/en/users/data/management/datahub/_index.md (hunk 19 above)
Comment (thebe14) [reply content/en/users/data/management/datahub/_index.md:1]: All should be like that. With details in subpages, neatly broken up by subject. So let's keep this, the others we'll improve in future PRs.

File: content/en/users/data/storage/_index.md (hunk 20 above)
Comment (thebe14) [reply content/en/users/data/storage/_index.md:33]: Nope. Table 4.1 (but not in table form) is on the parent page of this one.

File: content/en/users/aai/_index.md (hunk 7 above)
Comment (thebe14) [reply content/en/users/aai/_index.md:15]: Improved. But I do not know how to rewrite this in active voice, it's too generic.

File: content/en/users/compute/orchestration/im/cli/_index.md (hunk 15 above)
Comment (thebe14) [reply content/en/users/compute/orchestration/im/cli/_index.md:2]: Fixed

File: content/en/users/compute/orchestration/ec3/_index.md (hunk 14 above)
Comment (thebe14) [reply content/en/users/compute/orchestration/ec3/_index.md:3]: This is too long for the nav tree. Here users search for "what" not for the official name of a service. We have plenty of space for that in the actual page.

File: content/en/users/compute/orchestration/_index.md (hunk 13 above)
Comment (thebe14) [reply content/en/users/compute/orchestration/_index.md:27]: No, you are right. II added it to the table, maybe the description is not the best, though.

File: content/en/users/compute/cloud-compute/openstack/_index.md (hunk 12 above)
Comment (thebe14) [reply content/en/users/compute/cloud-compute/openstack/_index.md:80]: This page I did not touch. All pages under cloud compute that are not showing up in #411 will disappear in a future PR I am preparing.

File: content/en/users/data/_index.md (hunk 16 above)
Comment (andrea-manzi) [reply content/en/users/data/_index.md:17]: i would keep like this now, maybe to revisit when also openrdm doc is in

Comment (thebe14) [reply content/en/users/data/_index.md:17]: Added content for the openRDM page

**Comment by andrea-manzi** [comment]
we should remove draft: true from the openRDM page

**Comment by enolfc** [comment]
> I agree. That will get improved when the Getting Started part gets revised next. A lot of stuff about how to request access, the architecture, and more from D2.3 and D2.5 need to be added. But it is not what this PR should fix.
> 

Should we just remove the taskforce page in this PR? It's not bringing any value right now and I fear we will be postponing moving it forever. It simply does not belong to the docs.egi.eu but it's more a TCB-like activity and should be documented as such in confluence

**Comment by thebe14** [comment]
> Should we just remove the taskforce page in this PR? It's not bringing any value right now and I fear we will be postponing moving it forever.

Added draft flag, removed links to it.

**Review by enolfc — APPROVED** [review]
LGTM! Thanks @thebe14

**Pull request merged**